config/helm/aws-node-termination-handler/templates/_helpers.tpl

```diff
@@ -58,10 +66,19 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
 
 {{/*
-Create chart name and version as used by the chart label.
+Selector labels for the deployment
 */}}
-{{- define "aws-node-termination-handler.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- define "aws-node-termination-handler.selectorLabelsDeployment" -}}
+{{ include "aws-node-termination-handler.selectorLabels" . }}
+app.kubernetes.io/component: deployment
+{{- end -}}
```

Comment:
Please be mindful when changing immutable fields, it requires additional work when updating the helm chart to ensure high availability.

Reply:
@cainelli this work was intended to be a breaking change and these changes were fully intentional and made with the understanding of the implications. I think there might have been a some missed communication when this was released (see above) but technically all changes to a pre v1 SemVer package are breaking.

Reply:
Thanks for clarifying @stevehipwell and sorry for the grumpiness.